### DISCUSSION TITLE
Sharing my models!

### DISCUSSION THREAD

**Discussion by 38github** [GuitarML/PedalNetRT#38] [Ideas]
The names of the devices I have profiled have been changed so you have to make qualified guesses.

**Comment by 38github** [comment]
**Box Vaxurator** set at:

- More = On
- Pad = On
- Gain = 03:00
- Tone = 10:30
- Volume = 11:30
- Epoches: 7000 (not sure if I hear a difference from the 3000 version I have)

[box_vaxurator-more_pad_d3_t1030_v1130.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9748343/box_vaxurator-more_pad_d3_t1030_v1130.wav.zip)
[box_vaxurator-more_pad_d3_t1030_v1130_epoch7000_20221010.json.zip](https://github.com/GuitarML/PedalNetRT/files/9748357/box_vaxurator-more_pad_d3_t1030_v1130_epoch7000_20221010.json.zip)


It is alright to include it in the _SmartPedal Tones_ package if you would wish to do so.

**Comment by 38github** [comment]
**Mart Carlton CD Diver** set at:

- Mode = Reg
- Gain = 03:00
- Tone = 03:00
- Volume = 11:30
- Epoches: 3000

[mart_carlton_cd_diver-reg_d3_t3.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9748705/mart_carlton_cd_diver-reg_d3_t3.wav.zip)
[mart_carlton_cd_diver-reg_d3_t3_3000epoch_20221010.json.zip](https://github.com/GuitarML/PedalNetRT/files/9754410/mart_carlton_cd_diver-reg_d3_t3_3000epoch_20221010.json.zip)

It is alright to include it in the _SmartPedal Tones_ package if you would wish to do so.

**Comment by 38github** [comment]
**Mart Carlton CD Diver** set at:

- Mode = Fat (the earlier was _Reg_)
- Gain = 03:00
- Tone = 03:00
- Volume = 11:30
- Epoches: 3000

[mart_carlton_cd_diver-fat_d3_t3.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9754385/mart_carlton_cd_diver-fat_d3_t3.wav.zip)
[mart_carlton_cd_diver-fat_d3_t3_3000epoch_20221010.json.zip](https://github.com/GuitarML/PedalNetRT/files/9754355/mart_carlton_cd_diver-fat_d3_t3_3000epoch_20221010.json.zip)

Less CPU version but 95% of the quality retained:
[dc_drive_fat_drive_3_tone_3_3000epoch_20221010_less_cpu.json.zip](https://github.com/GuitarML/PedalNetRT/files/9780332/dc_drive_fat_drive_3_tone_3_3000epoch_20221010_less_cpu.json.zip)

It is alright to include it in the _SmartPedal Tones_ package if you would wish to do so.

**Comment by GuitarML** [comment]
@38github Yes here is great, you can also email them to "smartguitarml@gmail.com" but it may take me awhile to add them to the Tone Library. And thanks for the models! I'd like to have an automated model sharing website up at some point, but that's not my area of expertise. I may need to pass the torch on that one!

**Reply by 38github** [reply]
You are welcome and I am very thankful that you offer this software and as a bonus... open source!

**Comment by 38github** [comment]
**Zibane 8S0 Mini Buzz** set at:

- Sustain = 05:00 (full)
- Tone = 01:00
- Level = 11:00
- Epoches: 3000

[zibane_8s0_mini_buzz_sustain05_tone01_level11.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9757929/zibane_8s0_mini_buzz_sustain05_tone01_level11.wav.zip)
[zibane_8s0_mini_buzz_sustain05_tone01_level11.json.zip](https://github.com/GuitarML/PedalNetRT/files/9757928/zibane_8s0_mini_buzz_sustain05_tone01_level11.json.zip)

It is alright to include it in the _SmartPedal Tones_ package if you would wish to do so.

**Comment by 38github** [comment]
**Ty-Ranny Dumb Hunny** set at:

- Boost: Off
- Gain = 05:00 (full)
- Tone = 03:00
- Level = 10:00
- Epoches: 3000

[ty-ranny_dumb_hunny_boost_off_gain_0500_tone_0300_level_1000.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9761851/ty-ranny_dumb_hunny_boost_off_gain_0500_tone_0300_level_1000.wav.zip)
[ty-ranny_dumb_hunny_boost_off_gain_0500_tone_0300_level_1000_epoch_3000_20221011.json.zip](https://github.com/GuitarML/PedalNetRT/files/9761856/ty-ranny_dumb_hunny_boost_off_gain_0500_tone_0300_level_1000_epoch_3000_20221011.json.zip)


**Ty-Ranny Dumb Hunny** set at:

- Boost: On
- Gain = 05:00 (full)
- Tone = 03:00
- Level = 10:00
- Epoches: 3000

[ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000.wav.zip](https://github.com/GuitarML/PedalNetRT/files/9762331/ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000.wav.zip)
[ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000_epoch_3000_20221011.json.zip](https://github.com/GuitarML/PedalNetRT/files/9762332/ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000_epoch_3000_20221011.json.zip)

Less CPU version but 95% of the quality retained:
[ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000_epochs_3000_less_cpu.json.zip](https://github.com/GuitarML/PedalNetRT/files/9780300/ty-ranny_dumb_hunny_boost_on_gain_0500_tone_0300_level_1000_epochs_3000_less_cpu.json.zip)

It is alright to include these in the _SmartPedal Tones_ package if you would wish to do so.

**Comment by 38github** [comment]
**FRM Very Good Leveling Unit** set at:

Actual Values:
- Ratio = 6
- Attack = 2.5
- Release = 3
- Log Rel: On

[frm_rnla_very_good_leveling_unit_marker_2_e3500.zip](https://github.com/GuitarML/PedalNetRT/files/10071675/frm_rnla_very_good_leveling_unit_marker_2_e3500.zip)  (json file zipped)

The model is **very** accurate!

Colab learning data:

e100  = 0.136  loss
e200  = 0.12
e300  = 0.103
e400  = 0.0877
e500  = 0.0777
e600  = 0.0606

e1000 = 0.0167

e1500 = 0.00905

e2000 = 0.00715
e2100 = 0.0841 (increased and muffled the sound. Went on to go all the way up to ~0.750 loss before decreasing again)
e2200 = 0.0607
e2300 = 0.0422
e2400 = 0.0305
e2500 = 0.0272
e2600 = 0.0205
e2700 = 0.0189
e2800 = 0.0176
e2900 = 0.0196
e3000 = 0.016 (the muffled sound has disappeared somewhere between e2100 and now)
e3100 = 0.0143
e3200 = 0.014
e3300 = 0.0125
e3400 = 0.012
e3500 = 0.0114

It is alright to include it in the _SmartPedal Tones_ package if you would wish to do so.
